.github/workflows/ci.yml

```diff
@@ -38,7 +43,15 @@ jobs:
             **/setup.cfg
       - run: pip install -U pip wheel setuptools
       - run: pip install -U tox tox-gh-actions
-      - run: tox
+      - name: "Run tox targets for ${{ matrix.python-version }}"
+        env:
+          TOX_SKIP_ENV: ".*djangomain.*"
+        run: "python -m tox"
```

Comment:
I think this needs an `if: ${{ matrix.djangomain != "djangomain" }}` to avoid duplicate tests :)